tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/etc/EtcTypeChecker.scala

```diff
@@ -92,7 +92,7 @@ class EtcTypeChecker(varPool: TypeVarPool, inferPolytypes: Boolean = false) exte
             computeRec(ctx, solver, mkConst(ex.sourceRef, knownType))
           }
         } else {
-          onTypeError(ex.sourceRef, s"Undefined name $name. Introduce a type annotation.")
+          onTypeError(ex.sourceRef, s"Found no annotation for $name. Did you write one?")
```

Comment:
```suggestion
          onTypeError(ex.sourceRef, s"No annotation found for $name.")
```

Reply:
I am trying to give the user an action hint about what to do instead of just displaying an error.

Reply:
I think we are not used to helpful communication with tools. Trying to make the error messages actionable :-)

Reply:
I think it's obvious that if an annotation is missing, the way to fix it is to supply an annotation. The rhetorical questions don't help with communication.

If you want to give a hint, imo it should be something like: "Ensure the variable `$name` is annotated. See <link to docs> for more on type annotations.". Maybe it's just me, but I find these rhetorical questions irritating.

Reply:
I also find these questions annoying. But are we average users? :D

Reply:
I have changed to a less chatty message: "No annotation found for $name. Make sure that you've put one in front of $name."